src/Sulu/Bundle/AdminBundle/Resources/js/components/Select/OverlayListPositioner.js

```diff
@@ -0,0 +1,115 @@
+// @flow
+import type {OverlayListDimensions, OverlayListStyle} from './types';
+
+const MIN_HEIGHT = 100;
+const INDENT_FROM_ANCHOR = 5;
+const PADDING_TO_WINDOW = 10;
+
+type VerticalCrop = {
```

Comment:
This type is only needed in this class and doesn't even make sense outside of this class. This is just an alias to reduce the amount of characters in the type signatures.